manticore/platforms/linux.py

```diff
@@ -1744,12 +1760,14 @@ def sys_dup2(self, fd, newfd):
         try:
             file = self._get_fd(fd)
         except FdError as e:
-            logger.info("DUP2: fd ({fd}) is not open. Returning {-e.err}")
+            logger.info("sys_dup2: fd ({fd}) is not open. Returning -{errorcode(e.err)}")
             return -e.err
 
-        soft_max, hard_max = self._rlimits[self.RLIMIT_NOFILE]
+        soft_max, hard_max = self._rlimits[resource.RLIMIT_NOFILE]
```

Comment:
Note, there was a typo here previously, and the code would never have worked...  After I had added some type hints, and mypy started looking at this function, the typo was revealed.  Guess we don't have tests for this at present.